packages/kit/tsconfig.json

```diff
@@ -9,8 +9,10 @@
 		"moduleResolution": "node",
 		"allowSyntheticDefaultImports": true,
 		"paths": {
-			"types/*": ["./types/*"]
+			"test": ["./test/types"],
+			"types/*": ["./types/*"],
+			"@sveltejs/kit": ["../types"]
 		}
 	},
-	"include": ["src/**/*", "test/**/*"]
+	"include": ["src/**/*", "test/**/*", "types/**/*", "test/types.d.ts", "test/ambient.d.ts"]
```

Comment:
I didn't want to mix the `src` types and the `test` types up together, they feel like they belong in a separate place to me. It's better to `import('@sveltejs/kit')` rather than the internal types, because then we can verify that we're only using public interfaces (and that they're correct)